lib/chitter.rb

```diff
@@ -0,0 +1,34 @@
+require 'pg'
+
+class Chitter
+
+  attr_reader :id, :message, :text
+
+  def initialize(id:, message:, text:)
+    @id  = id
+    @message = message
+    @text = text
+  end
+
+  def self.all
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_manager_test')
+    else
+      connection = PG.connect(dbname: 'chitter_manager')
+    end
+    result = connection.exec("SELECT * FROM chitters")
+    result.map do |chitter|
+      Chitter.new(id: chitter['id'], message: chitter['message'], text: chitter['text'])
+    end
+  end
+
+  def self.create(text:, message:)
+    if ENV['ENVIRONMENT'] == 'test'
+    connection = PG.connect(dbname: 'chitter_manager_test')
+    else
+    connection = PG.connect(dbname: 'chitter_manager')
+    end
+  result = connection.exec("INSERT INTO bookmarks (text, message) VALUES('#{text}', '#{message}') RETURNING id, message, text;")
```

Comment:
THis code breaks the tests and the app. This is inserting into Bookmarks.. from the previous challenge